tests/benchmarks/android_geekbench.py

```diff
@@ -64,19 +64,17 @@ class GeekbenchTest(LisaBenchmark):
     bm_collect = 'ftrace energy'
 
     def benchmarkInit(self):
+        super(GeekbenchTest, self).benchmarkInit()
         self.setupWorkload()
         self.setupGovernor()
-        if self.reboot:
-            self.reboot_target()
```

Comment:
With the proposed modifications (e.g. properly placed private methods) we should both fix this horror (:heart:) as well as avoid the need to call the super-class implementation.